clitools

```diff
@@ -155,6 +155,47 @@ class DatabaseConnectionChecker extends Command {
     }
 }
 
+class CSVExport extends Command {
+    public function run() {
+        global $log;
+        list($slack_credentials, $caldav_credentials, $agenda_args) = read_config_file();
+        $caldav = new CalDAVClient($caldav_credentials['url'], $caldav_credentials['username'], $caldav_credentials['password']);
+        $salt = random_bytes(40);
+        $csv = "";
+        
+        foreach($caldav->fetchEvents(array_keys($caldav->getETags())) as $i => $event) {
+            
+            $vCalendar = \Sabre\VObject\Reader::read($event['vCalendarRaw']);
+            
+            $csv .= $vCalendar->VEVENT->DTSTART->getDateTime()->getTimestamp() . ";";
+            $csv .= $vCalendar->VEVENT->DTEND->getDateTime()->getTimestamp() . ";";
+
+            $attendees = [];
+            if(isset($vCalendar->VEVENT->ATTENDEE)) {
+                foreach($vCalendar->VEVENT->ATTENDEE as $attendee) {
+                    $attendees[] = hash_hmac("sha256", (string)$attendee, $salt);
```

Comment:
I'm not sure I get the point of having a hash given that the salt will be different at every run and the the csv file is appended (so a same user could correspond to different hashes.
Should we for instance just count the number of attendees? Or perhaps on the contrary we should not append the csv file? Or perhaps I misunderstand something?

Reply:
Actually the `FILE_APPEND` flag is superfluous (my bad). The idea is to export all events within one year and to give the produced file to the python script of PR #161 to generate plots. 
This script needs to identify attendees (along the year) so we can't just count the attendees.
I should have explain that sorry.